index.html

```diff
@@ -19,11 +21,26 @@
  <!-- navigation code here -->
 
   <div class="container home">
+    <div class="space-top">
+    <h1 class="top">SpaceWalkers</h1>
+    <nav class="my-mixin">
+      <a>Home</a>
+      <a>About Us</a>
+      <a>Blog</a>
+      <a>Contact</a>
+      <a>Sign in</a>
+    </nav>
+    </div>
 
     <header class="intro">
        <!-- image code here -->
+       <section class="earth-contain">
+       <div class="earth-above">
+         <img class="above" src="img/main-header.png">
+      </div>
```

Comment:
always remember to add in the alt attribute when you use img tags. we want to be accessible for all users, even those that use screen readers. the alt attribute is a short string that describes the image. am example of this could be something like <img class="above" src="img/main-header.png" alt="a picture of the earth from space" >